### PR TITLE
adds option to use 'get' instead of 'head' while scanning

### PR DESCRIPTION
**Pull request by katerberg**

### CODE REVIEW

File: lib/internal/checkUrl.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ function checkUrl(link, baseUrl, options, callback)
 	bhttp.request(link.url.resolved,  // TODO :: https://github.com/joepie91/node-bhttp/issues/3
 	{
 		headers: { "user-agent":options.userAgent },
-		method: "head"
```
Comment (stevenvachon) [inline lib/internal/checkUrl.js]: `requestMethod` might be more familiar

File: lib/internal/defaultOptions.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ var defaultOptions =
 	maxSockets: Infinity,
 	maxSocketsPerHost: 1,
 	rateLimit: 0,
+	retrievalVerb: 'head',
```
Comment (stevenvachon) [inline lib/internal/defaultOptions.js]: double quotes

File: README.md [hunk 3]
```diff
@@ -195,6 +195,11 @@ Type: `Number`
 Default value: `0`  
 The number of milliseconds to wait before each request.
 
+### options.retrievalVerb
+Type: `String`  
+Default value: `"head"`  
+The http verb to use in checking links. Some sites do not respond correctly to `"head"` so `"get"` can work better.
```
Comment (stevenvachon) [inline README.md]:
```markdown
The HTTP request method used in checking links. Some sites do not respond correctly to `"head"`, so `"get"` can provide more consistent and accurate results, albeit slower.
```

**Comment by stevenvachon** [comment]
And please squash commits (into this PR).

File: README.md [hunk 4]
```diff
@@ -195,6 +195,11 @@ Type: `Number`
 Default value: `0`  
 The number of milliseconds to wait before each request.
 
+### options.retrievalVerb
```
Comment (stevenvachon) [inline README.md]: options.requestMethod

**Comment by katerberg** [comment]
Closed in lieu of #18. I'm really not a fan of rewriting the history of this branch since someone else might have pulled it, so I just made a squashed commit up there.

**Comment by stevenvachon** [comment]
Ok. I'd thought that it convoluted the git history, but the last one you'd squashed this way looks as clean as rewriting history.

**Pull request closed without merging**